rskj-core/src/main/java/co/rsk/peg/BridgeEvents.java

```diff
@@ -99,6 +99,12 @@ public enum BridgeEvents {
                     new CallTransaction.Param(true, Fields.BTC_TX_HASH, SolidityType.getType(SolidityType.BYTES32)),
                     new CallTransaction.Param(false, "pegoutCreationRskBlockNumber", SolidityType.getType(SolidityType.UINT256))
             }
+    ),
+    PEGOUT_TRANSACTION_CREATED("pegout_transaction_created",
+        new CallTransaction.Param[]{
+            new CallTransaction.Param(true, Fields.BTC_TX_HASH, SolidityType.getType(SolidityType.BYTES32)),
+            new CallTransaction.Param(false, Fields.UTXO_OUTPOINT_VALUES, SolidityType.getType(SolidityType.BYTES))
+        }
     );
 
     private String eventName;
```

Comment:
A couple of ideas, to do in another task.

- Rename this enum to `BridgeEvent`. Singular, as enums should be
- Move this `Fields` class to an enum. Guess the same fields could be used for bridge events and bridge methods, right?
- Make the attributes of the class `final`

Reply:
Task created.